content/docs/command-reference/metrics/diff.md

```diff
@@ -9,7 +9,8 @@ commits in the <abbr>DVC repository</abbr>, or between a commit and the
 ```usage
 usage: dvc metrics diff [-h] [-q | -v]
                         [--targets [<path> [<path> ...]]]
-                        [-t <type>] [-x <path>] [-R] [--show-json]
+                        [-t <type>] [-x <path>] [-R]
+                        [--show-json] [--show-md]
                         [a_ref] [b_ref]
```

Comment:
Is this the actual order in which it's printed? I don't see this output in the core repo master branch yet. Thanks

Reply:
Yes, it wasn't merged then. Please take a look again.

Reply:
> Is this the actual order in which it's printed?

Yes, that's the order.

```sh
$ dvc metrics diff -h
usage: dvc metrics diff [-h] [-q | -v] [--targets [<paths> [<paths> ...]]] [-R] [--all] [--show-json] [--show-md]
                        [a_rev] [b_rev]

Show changes in metrics between commits
Documentation: <https://man.dvc.org/metrics/diff>

positional arguments:
  a_rev                 Old Git commit to compare (defaults to HEAD)
  b_rev                 New Git commit to compare (defaults to the current workspace)

optional arguments:
  -h, --help            show this help message and exit
  -q, --quiet           Be quiet.
  -v, --verbose         Be verbose.
  --targets [<paths> [<paths> ...]]
                        Metric files or directories (see -R) to show diff for. Shows diff for all metric files by default.
  -R, --recursive       If any target is a directory, recursively search and process metric files.
  --all                 Show unchanged metrics as well.
  --show-json           Show output in JSON format.
  --show-md             Show tabulated output in the Markdown format (GFM).
```

Note that we still need to readjust the docs for the `-t` and `-x` on the metrics. But, I have tried to keep the order for all `dvc {metrics,params} diff`.

Reply:
Looks like you're missing `[--all]` thouogh.

Reply:
I'll take care of it.